com.unity.ml-agents/Editor/RayPerceptionSensorComponentBaseEditor.cs

```diff
@@ -71,6 +71,7 @@ void UpdateSensorIfDirty()
         }
     }
 
+#if MLA_UNITY_PHYSICS2D_MODULE_ENABLED
```

Comment:
What happens if someone has a RayPerceptionSensorComponent3D added to their scene, then removes the module? We still need to get to RayPerceptionSensorComponent3D to validate the model shape. Maybe these should just display a ⚠️ message?

Reply:
good idea